site/testsuites/testsuite-javascript/__tests__/tbdex/issuer/credentialIssuance.test.js

```diff
@@ -0,0 +1,207 @@
+import { describe, test, expect, vi, beforeEach } from 'vitest';
+import express from 'express';
+import { Jwt, VerifiableCredential } from '@web5/credentials';
+
+const app = express();
+app.use(express.json());
+
+// :snippet-start: sanctionsCredentialClass
+class SanctionsCredential {
+  constructor(check) {
+    this.listsCleared = check.listsCleared;
+  }
+}
+
+export default SanctionsCredential;
+// :snippet-end:
+
+const issuerDid = {
+  did: 'did:key:z6MkgUZ55uLZGb4qcCLfZaJwCTXUxhxFJtKwkTaimigbaSqs',
+  document: {
+    'id': 'did:key:z6MkgUZ55uLZGb4qcCLfZaJwCTXUxhxFJtKwkTaimigbaSqs',
+    'verificationMethod': [
+      {
+        id: 'did:key:z6MkgUZ55uLZGb4qcCLfZaJwCTXUxhxFJtKwkTaimigbaSqs#z6MkgUZ55uLZGb4qcCLfZaJwCTXUxhxFJtKwkTaimigbaSqs',
+        type: 'JsonWebKey2020',
+        controller: 'did:key:z6MkgUZ55uLZGb4qcCLfZaJwCTXUxhxFJtKwkTaimigbaSqs',
+        publicKeyJwk: {
+          alg: 'EdDSA',
+          crv: 'Ed25519',
+          kty: 'OKP',
+          x: 'Hgw0rAhANmxjM6YLzhwN87hHM7PM0HUsMAUulE2W634',
+        },
+      },
+    ],
+    'authentication': [
+      'did:key:z6MkgUZ55uLZGb4qcCLfZaJwCTXUxhxFJtKwkTaimigbaSqs#z6MkgUZ55uLZGb4qcCLfZaJwCTXUxhxFJtKwkTaimigbaSqs',
+    ],
+    'assertionMethod': [
+      'did:key:z6MkgUZ55uLZGb4qcCLfZaJwCTXUxhxFJtKwkTaimigbaSqs#z6MkgUZ55uLZGb4qcCLfZaJwCTXUxhxFJtKwkTaimigbaSqs',
+    ],
+    'capabilityInvocation': [
+      'did:key:z6MkgUZ55uLZGb4qcCLfZaJwCTXUxhxFJtKwkTaimigbaSqs#z6MkgUZ55uLZGb4qcCLfZaJwCTXUxhxFJtKwkTaimigbaSqs',
+    ],
+    'capabilityDelegation': [
+      'did:key:z6MkgUZ55uLZGb4qcCLfZaJwCTXUxhxFJtKwkTaimigbaSqs#z6MkgUZ55uLZGb4qcCLfZaJwCTXUxhxFJtKwkTaimigbaSqs',
+    ],
+    '@context': [
+      'https://www.w3.org/ns/did/v1',
+      'https://w3id.org/security/suites/jws-2020/v1',
+    ],
+  },
+  keySet: {
+    verificationMethodKeys: [
+      {
+        publicKeyJwk: {
+          alg: 'EdDSA',
+          crv: 'Ed25519',
+          kty: 'OKP',
+          ext: 'true',
+          key_ops: ['verify'],
+          x: 'Hgw0rAhANmxjM6YLzhwN87hHM7PM0HUsMAUulE2W634',
+        },
+        privateKeyJwk: {
+          d: 'xEv-wUYjDOi-SahxPCoKyuh8aqTiV6CtEgUfHg3ys2Y',
```

Comment:
```suggestion
          d: '**********',
```

mask out private keys